<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JsonParser</class>
 <widget class="QMainWindow" name="JsonParser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2020</width>
    <height>1185</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>2020</width>
    <height>1185</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2020</width>
    <height>1185</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JsonParser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>441</width>
      <height>751</height>
     </rect>
    </property>
    <property name="title">
     <string>GetIn</string>
    </property>
    <widget class="QComboBox" name="languageChooseComboBox">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>40</y>
       <width>381</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Choose Language (Need &quot;Locales Dir Existed in {PWD}&quot;)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>zh_cn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>en_us</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="ReadJson">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Read ./data.json</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="p_JsonInputPlainTextEdit">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>110</y>
       <width>381</width>
       <height>561</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="p_GeneratedPushButton">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>680</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Parse</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_JsonTitleLabel">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>66</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Json</string>
     </property>
    </widget>
    <widget class="QComboBox" name="FileSelectComboBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Choose File</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>data.json</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>data2.json</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>1531</width>
      <height>751</height>
     </rect>
    </property>
    <property name="title">
     <string>OutPut</string>
    </property>
    <widget class="QLabel" name="l_ProvideLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Provide</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="p_SubScriptionPlainTextEdit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>330</width>
       <height>640</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="l_CFGsLabel">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>50</y>
       <width>42</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>CFGs</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="p_CFGsPlainTextEdit">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>90</y>
       <width>330</width>
       <height>640</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="l_SubScriptionLabel">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Subscription</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_ControlsLabel">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Controls</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="p_providePlainTextEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>330</width>
       <height>640</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>70</y>
       <width>321</width>
       <height>661</height>
      </rect>
     </property>
     <property name="title">
      <string>Layout</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>321</width>
        <height>639</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="uiGeneratedVLayout"/>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>860</y>
      <width>1981</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Debug Area</string>
    </property>
    <widget class="QLineEdit" name="signalDetailInfo">
     <property name="geometry">
      <rect>
       <x>1330</x>
       <y>40</y>
       <width>641</width>
       <height>23</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Click And Show Area</string>
     </property>
    </widget>
    <widget class="QLabel" name="DebugLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Debug</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="debugPlainTextEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>741</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QTreeWidget" name="signalTreeWidget">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>20</y>
       <width>471</width>
       <height>261</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="clearDebugMsgButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>20</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>1290</x>
       <y>50</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Argu:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1290</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="signalHostInfo">
     <property name="geometry">
      <rect>
       <x>1330</x>
       <y>80</y>
       <width>641</width>
       <height>23</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Click And Show Area</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1981</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QLabel" name="l_packageInfoDisplay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1961</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>%Display%</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2020</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
